lib/components/MegaMenuLayout/MegaMenuLayout.tsx

```diff
@@ -29,6 +30,7 @@ type Title = {
 const MegaMenuLayout: React.FC<MegaMenuWrapperProps> = ({
   tagline,
   title,
+  url,
```

Comment:
Can we also do the same for the `cmd+k` ?

Reply:
https://github.com/SSWConsulting/SSW.MegaMenu/pull/62